### ISSUE TITLE
NOMAD training

### ISSUE THREAD

**Issue by yz996to955** [robodhruv/visualnav-transformer#59]
Hello, author. I have two questions about NOMAD training that I would like to consult with you.
1、After training my NOMAD model on my own dataset, it did not show a multi-modal path distribution in intersection scenarios, but rather a uniform direction distribution. My training dataset consists of 1000 video clips, each with approximately 80 frames, and the distance between frames is 0.7m. What could be the reason for this? I set the mask to 1 during training, but the results still did not show a multi-modal distribution.
2、There is only one ground truth trajectory in front of the intersection; why does the trained model exhibit a multi-modal trajectory distribution?
Looking forward to your reply！

**Comment by zxp771** [comment]
您好，我想请问一下您具体是怎么训练NoMad模型的。是按照原始的diffusion policy的训练方式训练么，可是这样怎么输出轨迹呢？
